.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This was a little odd. If I remove `.prettierrc` entirely, when I save in VSCode it seems to be picking up a config file elsewhere on my system. That gives an inconsistent result from running `npx prettier --write .`. By including this empty config file, both format-on-save and explicitly running the Prettier command have the same output.

Reply:
makes sense, we need something here

Reply:
I guess my mental model was of per-setting overrides, but it does make sense that the entire configuration is one object.